docs/course_materials/exercises/07_Exceptions/homework.md

```diff
@@ -0,0 +1,24 @@
+# Exceptions - HomeWork 
+
+### ex1_exceptions.py
+The program asks for numeric number from the user input. 
```

Comment:
`
Create a new file ex1_exceptions.py 
Write a program that gets a number from the user, (use input function)
If the user doesn’t enter a numeric input the program normally would crash. 
Use try and except to catch the error  ValueError so that the program will not crush.
`